Makefile

```diff
@@ -162,7 +164,7 @@ receptorctl-test-venv/bin/pytest:
 	receptorctl-test-venv/bin/pip install -r receptorctl/test-requirements.txt
 
 receptorctl-tests: receptor receptorctl-test-venv/bin/pytest
-	cd receptorctl && ../receptorctl-test-venv/bin/pytest tests/tests.py
+	cd receptorctl && ../receptorctl-test-venv/bin/pytest --junitxml=../receptorctl-test-junit.xml tests/tests.py
```

Comment:
Perhaps this receptorctl-test-junit.xml should also be saved as an artifact from the GitHub workflow?

Reply:
Also, do receptorctl errors here get picked up by GitHub and reported in the GUI?  And are there relative paths in the file that would be broken by moving the file up a level?

Reply:
yes the receptorctl-test-junit.xml will be saved as an artifact, it just ends up in the same zip/tar as the other logs. Also yes the thing that reports the junit in the GUI will report the receptorctl test junit as well. Not sure about the relative paths but i expect you're right, i should probably fix that by removing the cd